src/attributes/tokens.ts

```diff
@@ -108,7 +108,7 @@ export const TokensAttributes: { [tokenSymbol: string]: ITokenAttributes } = {
   },
   WBTC: {
     symbol: 'WBTC',
-    logoURL: 'https://assets.coingecko.com/coins/images/7598/small/wrapped_bitcoin_wbtc.png',
+    logoURL: 'https://bscscan.com/token/images/btcb_32.png',
```

Comment:
The correct WBTC icon is the one from coingeko 
```suggestion
    logoURL: 'https://assets.coingecko.com/coins/images/7598/small/wrapped_bitcoin_wbtc.png',
```

Reply:
Done!